src/dotnet/commands/dotnet-run/RunCommandParser.cs

```diff
@@ -3,6 +3,7 @@
 
 using System.Collections.Generic;
 using System.Linq;
+using System.Reflection.Metadata;
```

Comment:
Nit: was this added for any particular reason?

Reply:
cleaned